BUILD.gn

```diff
@@ -779,6 +779,16 @@ if (is_mac) {
   source_set("electron_lib_arc") {
     include_dirs = [ "." ]
     sources = [
+      "//chrome/browser/extensions/global_shortcut_listener_mac.h",
```

Comment:
would be good to have `chrome_lib_arc` source_set that is restricted to sources in `chromium_src/BUILD.gn`

Reply:
Good idea - split these out into a separate "chrome_lib_arc" source_set with the needed deps and sources, and removed the unneeded deps/sources from "electorn_lib_arc"